GPflow/gpflowrc

```diff
@@ -15,3 +15,6 @@ ekern_quadrature = warn
 [profiling]
 dump_timeline = False
 dump_tensorboard = False
+output_file_name = timeline
```

Comment:
@markvdw Would this filename only be for the profiler? Maybe profiler_file_name etc would be less ambiguous.